lib/index.js

```diff
@@ -154,6 +163,51 @@ function filter(facade, list){
 }
 
 /**
+ * Format payload for each product.
+ *
+ * @param {Track} track
+ * @return {Array}
+ * @api private
+ */
+
+function formatItems(track){
+  return foldl(function(payloads, props){
+    var product = new Track({ properties: props });
```

Comment:
Mainly to use the helper functions which is what's being used in the `formatProducts` function so figured I'd keep it consistent but yeah probably don't need to.